pkg/api/handlers/generic/containers.go

```diff
@@ -57,6 +57,10 @@ func ListContainers(w http.ResponseWriter, r *http.Request) {
 	}{
 		// override any golang type defaults
 	}
+
+	// Default Size to false
+	query.Size = false
```

Comment:
Is this necessary? false is the default for booleans

Reply:
your right it's not necessary, should i withdraw the pr change and resubmit?

Reply:
You can just remove the code and amend your PR and force push it.

Reply:
Removed unneeded code